Dockerfile

```diff
@@ -36,6 +36,6 @@ WORKDIR /app
 USER app
 
 ENTRYPOINT ["/sbin/tini", "--"]
-CMD ["node", "start.js"]
+CMD ["node", "--max-http-header-size=32768", "--max-old-space-size=1300", "start.js"]
```

Comment:
what's `max-old-space-size`?

Reply:
When adding header size and running the server in local, getting old space error in console. So, added it.

Reply:
could you pls check what it does? and why setting it to 1300?
we gotta check what we're adding else it might cause more problems

Reply:
It is for memory management in the container. Sure. Will check